### ISSUE TITLE
Package not available

### ISSUE THREAD

**Issue by samvv** [elm-lang/url#4]
I can't seem to install the package, getting the following error:

```
Error: Could not find any packages named elm-lang/url.

Here are some packages that have similar names:

    elm-lang/core
    elm-lang/dom
    elm-lang/html
    elm-lang/svg
```

Is this normal? If not, is it possible to publish the package?

**Comment by jhrcek** [comment]
@samvv If you look at the history of this repo you might notice this package originated from https://github.com/evancz/url-parser and then underwent some changes in preparation for the upcoming release of Elm 0.19. I believe it will only be published once Elm 0.19 comes out. In the meantime you can still use evancz/url-parser with Elm 0.18

**Comment by evancz** [comment]
Yeah, this is not published yet. It'll be published with 0.19.

**Comment by alxvallejo** [comment]
How is it obvious that this isn't published?

**Comment by jhrcek** [comment]
@alxvallejo It's normally available on package.elm-lang.org: https://package.elm-lang.org/packages/elm/url/latest/  It was published around the time of release of elm 0.19
